### ISSUE TITLE
0.3.3+git20151209~webupd8 incorrectly displays some fonts

### ISSUE THREAD

**Issue by worldowner** [i-rinat/freshplayerplugin#296]
System is Ubuntu 14.04 LTS. After todays upgrade I have:

0.3.3+git20151209~webupd8~trusty and flash player 20.0.0.228 (distrubuted with Chrome).
Since this upgrade vcenter displays fonts in wrong way:
![freshplugin-vcenter](https://cloud.githubusercontent.com/assets/10338079/11775978/e2b68186-a243-11e5-89e6-dbb964426977.png)

The same page opened in Google Chrome works fine.

**Comment by i-rinat** [comment]
That was fixed in 8f5e6beea4d531c7172c7a65cd2e2e3eeae5ff4b.

There are two similar interfaces in PPAPI for font-related operations, and both have the same method available. PepperFlash was using one before, but switched to another in newer version.

**Comment by worldowner** [comment]
Thank you for information.

**Comment by storm9c1** [comment]
Hi, I am seeing the same broken text as shown above.  I am also running "freshplayerplugin       0.3.3+git20151006~webupd8~precise".  I was wondering when this fixed version will make it into the official webupd8 Ubuntu PPA?  Or should I open a bug through Launchpad?  Thanks!

**Comment by i-rinat** [comment]
> Or should I open a bug through Launchpad?

I think a bug on Launchpad won't make any harm. As far I know, @hotice did the packaging. Perhaps, he is busy or on vacation.

By the way, building a new version from source should not be very hard, although you'll need to install a number of dev packages. There are instructions in README file.

**Comment by vdanjean** [comment]
It seems fixed in 0.3.4 (if I'm not mistaken, the commit 8f5e6be is in the 0.3.4 tag)
I uploaded 0.3.4 in Debian unstable a few days ago. It reaches testing today. So I uploaded to jessie-backports a few minutes ago.
I do not know if the Ubuntu package is just synchronized with the Debian one or if a Ubuntu maintainer packages it separately. And I do not know the Ubuntu policy about updating packages in 14.04 LTS. Ask Ubuntu guys for that.
